### DISCUSSION TITLE
Notation CLI custom plugin path

### DISCUSSION THREAD

**Discussion by greatestfen** [notaryproject/notaryproject#316] [Q&A] [answered]
Hi there!
I'm trying to implement image signing to a pipeline and facing an issue with custom $HOME path in the runner configuration.
The $HOME variable is set to /tmp, so Notation throws the following error:
`Error: plugin executable file is either not found or inaccessible: stat /tmp/.config/notation/plugins/com.amazonaws.signer.notation.plugin/notation-com.amazonaws.signer.notation.plugin: no such file or directory`
Is it possible to point to the correct plugin directory when executing **notation** command? Or how to handle this?
Any help is very much appreciated.

**Comment by priteshbandi** [comment]
@greatestfen Where are you running notation? How did you install notation and aws-signer plugin?

**Reply by greatestfen** [reply]
@priteshbandi 
I run the image with preinstalled AWS CLI, notation, and aws-signer plugin. All of that is stored within the /root/.config/notation catalog.

**Comment by greatestfen** [comment, marked as answer]
We can close this conversation.
I solved this issue by setting up environment variables:
```
export NOTATION_LIBEXEC=/root/.config/notation
export NOTATION_CONFIG=/root/.config/notation
```
Thanks everyone!
